src/app/Plans/components/Wizard/GeneralForm.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import { Form, FormGroup, TextArea, TextInput, Title } from '@patternfly/react-core';
+import SimpleSelect, { OptionWithValue } from '@app/common/components/SimpleSelect';
+import { Provider } from '@app/Providers/types';
+
+interface IGeneralFormComponentProps {
+  sourceProviders: Provider[];
+  targetProviders: Provider[];
+}
+
+const GeneralForm: React.FunctionComponent<IGeneralFormComponentProps> = ({
+  sourceProviders,
+  targetProviders,
+}: IGeneralFormComponentProps) => {
+  const [planName, setPlanName] = React.useState<string>('');
+  const [planDescription, setPlanDescription] = React.useState<string>('');
+  const [sourceProvider, setSourceProvider] = React.useState<Provider | null>(null);
+  const [targetProvider, setTargetProvider] = React.useState<Provider | null>(null);
+
+  const sourceProvidersOptions = Object.values(sourceProviders).map((provider) => ({
```

Comment:
We can probably move these provider options / select stuff to a common `<ProviderSelect>` or something eventually, just making a note to myself for that. I have similar code in #24 / #25.